src/main/java/org/hmhb/kml/KmlDocument.java

```diff
@@ -0,0 +1,93 @@
+package org.hmhb.kml;
```

Comment:
@mckoon There are alot of Kml DTO objects in here (for the JAXB).  Do you think I should move them into a subpackage to decrease the noise in this directory?